docs/helpers/Appium.md

```diff
@@ -121,161 +114,199 @@ let browser = this.helpers['Appium'].browser
 
 -   `config`  
 
-### _switchToContext
+### runOnIOS
```

Comment:
Is there any reason to remove alphabetical order  from doc?

Reply:
I had to write a custom method aggregation (see runio.js) changes to merge methods from WebDriver and Appium classes. This was part of this.